common/sass/variables.scss

```diff
@@ -18,6 +18,7 @@ $black: #000;
 $black-lighter: #262B2F;
 $green: #D5E20F;
 $white-green: #F6FAF3;
+$white-almost: #f7f8f9;
```

Comment:
this color is already used in common.scss (`$lightgray: #f7f8f9;`) Could you remove duplication? probably common.scss should use your new defined color. Also please, make it uppercase as others are also uppercased (i am taking care of `off-white` on my pr)

Reply:
I didn't make it uppercase as according to sass linter it should be lowercase. In separate PR I will fix whole file to follow litner rules